frontends/web/src/locales/en/app.json

```diff
@@ -1134,12 +1135,16 @@
     "feeTarget": {
       "description": {
         "economy": "4 hours (24 blocks)",
+        "economy_eth": "< 30 minutes",
```

Comment:
might be misleading. I would spell it out: "30 minutes or less"

Reply:
done